src/Chameleon.php

```diff
@@ -186,3 +186,11 @@ protected function getThemeFilePath() {
 		return $GLOBALS[ 'egChameleonThemeFile' ];
 	}
 }
+
+// Support MediaWiki 1.39 and 1.44. Remove this when dropping support for 1.39.
+if ( version_compare( MW_VERSION, '1.44', '>=' ) ) {
+	class_alias( \MediaWiki\Html\Html::class, 'Html' );
```

Comment:
One could also instruct people to add those lines to their LocalSettings.php

Reply:
Also one could link to the full list https://gerrit.wikimedia.org/r/c/mediawiki/core/+/1124809